test-harness/specs/default_responses-AC-8.oas3.txt

```diff
@@ -0,0 +1,22 @@
+====test====
+When I send a request to an operation
+And in the path I specify ?__code=404
+And the operation has 404 status code response defined
+And in this operation there is no schema or example defined in the response
+Then I get back a 404 response with an error message
+====spec====
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      responses:
+        404:
+          description: no resource
+====server====
+mock -p 4010
+====command====
+curl -i -X GET http://localhost:4010/todos?__code=404
+====expect====
+HTTP/1.1 404 Not Found
+content-type: application/problem+json
```

Comment:
@philsturgeon I think this is incorrect. If there's a response with no content, we should return an empty text/plain, right?